### DISCUSSION TITLE
[Bug report] *.HEIC images support (Apple)

### DISCUSSION THREAD

**Discussion by monoista** [GetPublii/Publii#1583] [Bug Report]
### Operating system

MacOS 11.7.10

### Publii version

0.45.2 (0.45.2)

### Issue type

Application

### Bug description

Images of *.HEIC format are not being displayed and seems that not being supported (?)

**Comment by candidexmedia** [comment]
That image format isn't well supported for most users and browsers:

https://caniuse.com/?search=HEIC

Publii is an Electron app, and Electron is based on the Chromium browser, which doesn't support that format.
